pubspec.yaml

```diff
@@ -1,7 +1,6 @@
 name: http_mock_adapter
 description: A new Flutter package project.
```

Comment:
description მგონი ჯობია დავწეროთ რამე. any idea?

@campfire5 @GiorgiBeriashvili @nataliehippe ?

Reply:
აქ general description არ იწერება ხო?

Reply:
არ ვიცი, მგონი კი (?) @AlexGorgadz3

Reply:
პატარა აღწერა რომ გავუკეთოთ,ესე : პექიჯის სახელი for და რისთვისაც იქმნება